samples/android/app/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-android-extensions'
+    id 'kotlin-kapt'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
+
+    defaultConfig {
+        applicationId "org.hyperledger.indy.sdk"
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+}
+
+dependencies {
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'com.google.android.material:material:1.2.1'
+    implementation 'junit:junit:4.13.1'
+    testImplementation 'junit:junit:4.13.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    implementation files('libs/libindy-android-debug.aar')
+    implementation "androidx.constraintlayout:constraintlayout:2.0.2"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.9"
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0'
+
+    // toasty
+    implementation 'com.github.GrenderG:Toasty:1.5.0'
+
+
+    // java
+    api("org.slf4j:slf4j-api:1.7.30")
+    api("net.java.dev.jna:jna:5.6.0@aar")
+    api("org.apache.commons:commons-lang3:3.11")
+    api("net.sourceforge.streamsupport:android-retrofuture:1.7.2") {
+        because("We want to support api level 21.")
+    }
+    api("commons-io:commons-io:2.7")
+
+    // logger
+    implementation('com.github.bright:slf4android:0.1.6'){
+        transitive = true
+    }
+}
```

Comment:
set ide to add a new line at the end of the file when saved. You have this issue all over the place.